public/scripts/services/wordService.js

```diff
@@ -0,0 +1,44 @@
+angular.module('scrambleApp').
+  factory('wordFactory', ['$http', function($http) {
+
+    //Shuffle function for words
+    var shuffle = function(word) {
+      var randomIndex, temp;
+      for (var i = 0; i < word.length; i++) {
+        randomIndex = Math.floor(Math.random() * (word.length - i)) + i;
+        temp = word[i];
+        word[i] = word[randomIndex];
+        word[randomIndex] = temp;
+      }
+      return word;
+    }
+
+    return {
+      //Takes a callback and passes in a split random word from the API
+      getRandomWord: function(cb) {
+        $http.get('/randomWord').
```

Comment:
might want to do a RESTFUL Api, or at least have the endpoint hit  '/random-word'